src/test/scala/tech/cryptonomic/conseil/tezos/DatabaseConversionsTest.scala

```diff
@@ -776,14 +776,17 @@ class DatabaseConversionsTest
 
       "convert a Proposals operation to a database row" in {
 
-        val converted = (block, groupHash, Proposals: Operation).convertTo[Tables.OperationsRow]
+        val converted = (block, groupHash, sampleProposals: Operation).convertTo[Tables.OperationsRow]
 
         converted.operationId shouldBe 0
         converted.operationGroupHash shouldBe groupHash.value
         converted.blockHash shouldBe block.data.hash.value
         converted.blockLevel shouldBe block.data.header.level
         converted.timestamp shouldBe Timestamp.from(block.data.header.timestamp.toInstant)
         converted.kind shouldBe "proposals"
+        converted.source shouldBe Some(ContractId("tz1VceyYUpq1gk5dtp6jXQRtCtY8hm5DKt72"))
+        //converted.period shouldBe Some(10)
+        converted.proposal shouldBe Some("[\"Psd1ynUBhMZAeajwcZJAeq5NrxorM6UCU4GJqxZ7Bx2e9vUWB6z\"]")
```

Comment:
I would suggest using triple-quotes for raw strings (so we won't need to escape quotes)
`"""["hashcode"]"""`

Reply:
Good idea!